tests/integration/create.bats

```diff
@@ -81,3 +81,36 @@ function teardown() {
 
 	testcontainer test_busybox running
 }
+
+@test "runc create [second createRuntime hook fails]" {
+	update_config '.hooks |= {"createRuntime": [{"path": "/bin/true"}, {"path": "/bin/false"}]}'
+
+	runc create --console-socket "$CONSOLE_SOCKET" test_hooks
+	[ "$status" -ne 0 ]
+	[[ "$output" == *"error running createRuntime hook #1:"* ]]
+}
+
+@test "runc create [hook fails]" {
+	for hook in prestart createRuntime createContainer; do
+		echo "testing hook $hook"
+		# shellcheck disable=SC2016
+		update_config '.hooks |= {"'$hook'": [{"path": "/bin/true"}, {"path": "/bin/false"}]}'
+		runc create --console-socket "$CONSOLE_SOCKET" test_hooks
+		[ "$status" -ne 0 ]
+		[[ "$output" == *"error running $hook hook #1:"* ]]
+	done
+}
+
+@test "runc run [hook fails]" {
```

Comment:
Why not move these 3 test cases you added to `hooks.bats`? Especially this one should not be in here.

Reply:
ping @kolyshkin 
I think it will be more clear for code readers.

Reply:
@lifubang the reason is, `hooks.bats` has some non-trivial setup and cleanup, which my tests don't need, and limited requirements (`root` and `no_systemd`), which my tests don't have.

So, I can move hooks.bats to hooks_so.bats (as it tests hooks in form of `.so`), and move my tests into hooks.bats.

Reply:
@lifubang done; PTAL